test/test_support.ts

```diff
@@ -43,15 +43,20 @@ export const compilerOptions: ts.CompilerOptions = {
   outDir: '.',
 };
 
+/** Compiler options to use if you want to produce .d.ts outputs. */
+export const dtsCompilerOptions: ts.CompilerOptions = {
```

Comment:
Could you instead keep these options local to where they are used, i.e. in the dts test file?